extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -36,134 +33,170 @@ public static function onBeforePageDisplay() {
 
 	/**
 	 * Checks arguments for errors.
+	 * @param $modelData
 	 * @param array $args
-	 * @param string errorMessage Return parameter with the proper error message to show. Disregard if return is false
+	 * @internal param $errorMessage /Return parameter with the proper error message to show.
+	 * Disregard if return is false.
 	 * @return true if ok, false if error
 	 */
-	private function checkArguments( array $args, $modelData, &$errorMessage ) {
+	private function checkArguments( $modelData, array $args, &$errorMessage ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)